### DISCUSSION TITLE
NOT work?

### DISCUSSION THREAD

**Discussion by aviatordestructivo** [community/community#116676] [Repositories]
### Select Topic Area

Show & Tell

### Body

Hello, how are you, sorry I'm writing to you but I need to know how to make it work https://github.com/Preeaaw/Aviator-Predictor-FULL
It turns out that I opened it with Windows VSCode, but I don't know how to proceed. If you don't mind, could you give me a hand brother...? thank you

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by aviatordestructivo** [reply]
Oh, sorry, sorry, I don't want to bother....where could I look for the aviator theme???? I haven't found a place...thanks I'm new
